lib/webpacker.rb

```diff
@@ -2,14 +2,15 @@ module Webpacker
   extend self
 
   def bootstrap
-    Webpacker::Env.load
-    Webpacker::Configuration.load
-    Webpacker::Manifest.load
+    Webpacker::Env.load_instance
+    Webpacker::Configuration.load_instance
+    Webpacker::DevServer.load_instance
+    Webpacker::Manifest.load_instance
   end
 
   def compile
     Webpacker::Compiler.compile
-    Webpacker::Manifest.load
+    Webpacker::Manifest.load_instance
```

Comment:
Sorry but could we please drop these changes and move to another PR that way we can merge this one quickly and then address the naming issue in another PR :) ?